ADO.NET/ResultSetPrinter.cs

```diff
@@ -2,65 +2,103 @@
 using System.Configuration;
 using System.Data.Common;
 using Vertica.Data.VerticaClient;
+using System.Collections.ObjectModel;
+using System.Linq;
 
-class ResultSetPrinter {
+internal class ResultSetPrinter {
+        private int m_maxColumnWidth = Int32.Parse (ConfigurationManager.AppSettings["MaxColumnWidth"]);
+        private int m_widestColumnRead = 0;
+        private VerticaDataReader m_reader;
+        private ReadOnlyCollection<DbColumn> m_columns;
+        private const string TRUNCATION_SYMBOL = "...";
+        private bool m_ValuesTruncated = false;
 
-    private static int maxColumnWidth = Int32.Parse (ConfigurationManager.AppSettings["MaxColumnWidth"]); 
-    
-    public void printResults (VerticaDataReader reader) 
-    {        
-        var columns = reader.GetColumnSchema();
-        printColumnHeader (columns);
+        public ResultSetPrinter (VerticaDataReader reader) {
+            m_reader = reader;
+            m_columns = m_reader.GetColumnSchema();
+        }
 
-        // Write the data
-        while (reader.Read())
+        public void printResults () 
         {
-            for (int i = 0; i < reader.FieldCount; i++)
+            printColumnHeaders ();
+
+            // Write the data
+            while (m_reader.Read())
             {
-                int columnWidth = getPrintedColumnWidth (columns[i]);
-                string Value = reader.GetValue(i).ToString ();
-                Console.Write("{0} ", Value.PadRight (columnWidth));
+                for (int i = 0; i < m_reader.FieldCount; i++)
+                {
+                    string Value = m_reader.GetValue(i).ToString ();
```

Comment:
nit: remove extra space after ToString

Reply:
There's a lot of places where there's a space between the function and parameters. That's not conventional style, but it's fine. It might also just be a git formatting thing.

Reply:
I changed both ResultSetPrinter and SampleApp to conform with the convention.